dockerfiles/dev/Dockerfile

```diff
@@ -23,21 +23,14 @@ FROM porepy/base
 ## Download PorePy from github
 # Make sure we are in the home directory
 WORKDIR ${POREPY_HOME}
-USER porepy
-# Clone from Github
-RUN git clone https://github.com/pmgbergen/porepy.git ${POREPY_SRC}
-
-
-# Enter source directory
-WORKDIR ${POREPY_SRC}
 # Make sure we're on the develop branch
 RUN git checkout develop
 
 # Do the install
 RUN pip install -e .
```

Comment:
Why not install in base file?

Reply:
To avoid a double install when building the stable version. It is a marginal difference (at least for something as simple as PorePy), and one can think of both options as conceptually simpler. Basically, I ended up using the same workflow as fenics applies, on the assumption they know what they are doing.

Reply:
Do I get this right: if you install, then check out a different branch, there's some sort of re-install going on?

Reply:
Not really. Install of PorePy proper is done in the relevant front image (dev or stable), while base prepares the ground. If you check out, PorePy would be updated since we install with the -e option (which could be taken as a sign we should not use that option in stable, but then, playing with git in the stable docker image is far from intended usage).

There are probably several ways one can do this, but this is something that just should work.